workspaces/services/guacamole-azure-dsvm/terraform/dsvm.tf

```diff
@@ -0,0 +1,128 @@
+  access                      = "Deny"
+  destination_address_prefix  = "*"
+  destination_port_range      = "*"
+  direction                   = "Inbound"
+  name                        = "deny-inbound-override"
+  network_security_group_name = azurerm_network_security_group.dsvmnsg.name
+  priority                    = 900
+  protocol                    = "*"
+  resource_group_name         = data.azurerm_resource_group.ws.name
+  source_address_prefix       = "*"
+  source_port_range           = "*"
+}
+
+resource "azurerm_network_security_rule" "deny-outbound-override-dsvm" {
+  access                      = "Deny"
+  destination_address_prefix  = "*"
+  destination_port_range      = "*"
+  direction                   = "Outbound"
+  name                        = "deny-outbound-override"
+  network_security_group_name = azurerm_network_security_group.dsvmnsg.name
+  priority                    = 4096
+  protocol                    = "*"
+  resource_group_name         = data.azurerm_resource_group.ws.name
+  source_address_prefix       = "*"
+  source_port_range           = "*"
+}
+
+resource "azurerm_network_interface_security_group_association" "dev_vm_nsg_association" {
+  network_interface_id      = azurerm_network_interface.internal.id
+  network_security_group_id = azurerm_network_security_group.dsvmnsg.id
+}
+
+resource "random_string" "username" {
+  length      = 4
+  upper       = true
+  lower       = true
+  number      = true
+  min_numeric = 1
+  min_lower   = 1
+  special     = false
+}
+
+resource "random_password" "password" {
+  length           = 16
+  lower            = true
+  min_lower        = 1
+  upper            = true
+  min_upper        = 1
+  number           = true
+  min_numeric      = 1
+  special          = true
+  min_special      = 1
+  override_special = "_%@"
+}
+
+resource "azurerm_virtual_machine" "dsvm" {
+  name                             = local.vm_name
+  location                         = data.azurerm_resource_group.ws.location
+  resource_group_name              = data.azurerm_resource_group.ws.name
+  network_interface_ids            = [azurerm_network_interface.internal.id]
+  vm_size                          = "Standard_DS1_v2"
+  delete_os_disk_on_termination    = false
+  delete_data_disks_on_termination = false
+
+  storage_image_reference {
+    publisher = "microsoft-dsvm"
+    offer     = "dsvm-windows"
+    sku       = "server-2016"
+    version   = "latest"
+  }
+
+  storage_os_disk {
+    name              = "osdisk-${local.vm_name}"
+    caching           = "ReadWrite"
+    create_option     = "FromImage"
+    managed_disk_type = "Standard_LRS"
+  }
+
+  os_profile {
+    computer_name  = local.vm_name
+    admin_username = random_string.username.result
+    admin_password = random_password.password.result
+  }
+
+  os_profile_windows_config {
+  }
+
+  identity {
+    type = "SystemAssigned"
+  }
+
+  tags = {
+    parent_service_id = var.guacamole_parent_service_id
```

Comment:
Are you doing anything with this apart from the Tags?

Reply:
no. its more of a 'todo' item for me to make sure that this bundle install does get the parent id .